lib/changes/dedentLines.js

```diff
@@ -29,7 +29,7 @@ function dedentLines(
     const { document, selection } = value;
     const lines = document
         .getBlocksAtRange(selection)
-        .filter(node => node.type === opts.lineType);
+        .filter(node => node.type === opts.get('lineType'));
```

Comment:
Why do we need to use `.get` now ?

Reply:
I saw you comment that it returns undefined... but is it because of the upgrade from `3.8.1` to `3.8.2` ? According to Immutable's changelog, nothing actually changed, just the license. https://github.com/facebook/immutable-js/releases/tag/v3.8.2

Reply:
That was weird, `.xxx` just works now.
I copied the code into a react app and fix it, so I can see the actual result on the browser.

Maybe the version of immutable in my project is different.
Anyway, restored the `.xxx` way now.